client/my-sites/hosting/web-server-settings-card/index.js

```diff
@@ -189,6 +215,51 @@ const WebServerSettingsCard = ( {
 		);
 	};
 
+	const getWpVersionContent = () => {
+		if ( isGettingWpVersion ) {
+			return;
+		}
+
+		const isWpVersionButtonDisabled =
+			disabled || ! selectedWpVersion || selectedWpVersion === wpVersion;
+		const selectedWpVersionValue =
+			selectedWpVersion || wpVersion || ( disabled && recommendedValue );
+
+		return (
+			<FormFieldset>
+				<FormLabel>{ translate( 'WordPress version' ) }</FormLabel>
+				<FormSelect
+					disabled={ disabled || isUpdatingPhpVersion }
+					className="web-server-settings-card__wp-version-select"
+					onChange={ ( event ) => setSelectedWpVersion( event.target.value ) }
+					value={ selectedWpVersionValue }
+				>
+					{ getWpVersions().map( ( option ) => {
+						return (
+							<option
+								disabled={ option.value === phpVersion }
+								value={ option.value }
+								key={ option.label }
+							>
+								{ option.label }
+							</option>
+						);
+					} ) }
+				</FormSelect>
+				{ ! isWpVersionButtonDisabled && (
+					<Button
+						className="web-server-settings-card__wp-set-version"
```

Comment:
This class just needs a little `margin-top`, like `.web-server-settings-card__php-set-version`

Reply:
I noticed this too, we should make both buttons consistent:

![Screenshot 2023-08-01 at 12 19 53](https://github.com/Automattic/wp-calypso/assets/727413/2ac14fcd-2167-4d49-ab14-57da7b369dc6)

Reply:
Fixed with 245d10efbf45677019fd8405af95004bbc837259